dspace-spring-rest/src/main/java/org/dspace/app/rest/model/PoolTaskRest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.dspace.app.rest.RestResourceController;
+
+/**
+ * The PoolTask REST Resource
+ *
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ */
+public class PoolTaskRest extends BaseObjectRest<Integer> {
+    public static final String NAME = "pooltask";
+    public static final String CATEGORY = RestAddressableModel.WORKFLOW;
+
+    private String step;
+
+    private String action;
+
+    @JsonIgnore
+    private WorkflowItemRest workflowitem;
+
+    @Override
+    public String getCategory() {
+        return CATEGORY;
+    }
+
+    @Override
+    public String getType() {
+        return NAME;
+    }
+
+    @Override
+    public Class getController() {
+        return RestResourceController.class;
+    }
+
+    public String getStep() {
```

Comment:
Please add Javadocs to all, new public methods (which are not override)

Reply:
it is just a DTO mapping the data model class, add link to the corresponding data model method/attribute https://github.com/DSpace/DSpace/pull/2312/files#diff-8afc90b30c652b5b004a16ef220c8124R46